e2e-tests/IIoTPlatform-E2E-Tests/Standalone/B_PublishMultipleNodesStandaloneDirectMethodTestTheory.cs

```diff
@@ -170,6 +189,9 @@ IIoTMultipleNodesTestContext context
             // Make sure that there is no active monitoring.
             await TestHelper.StopMonitoringIncomingMessagesAsync(_context, cts.Token);
 
+            // Clean publishednodes.json.
+            await TestHelper.PublishNodesAsync(Array.Empty<PublishedNodesEntryModel>(), _context);
```

Comment:
Can we please also have test cases for multiple endpoints ? This would make sure that we don't get mixed results for different endpoints. This can be `C_PublishMultipleEndpointsStandaloneDirectMethodTestTheory.cs` suite, for example.

Reply:
add new test case for multiple endpoints